chromium_src/chrome/browser/resources/tab_search/auto_tab_groups/auto_tab_groups_page.ts

```diff
@@ -23,6 +23,8 @@ export class AutoTabGroupsPageElement extends CrLitElement {
   protected topics_: string[] = []
   protected topic = ''
   protected undoTopic_ = ''
+  protected showFRE_: boolean =
```

Comment:
nit: `boolean` is implied
```suggestion
  protected showFRE_ =
```

Reply:
Removed